db/seeds.rb

```diff
@@ -27,6 +27,104 @@
 GradeLevel.create "title" => "College Senior", "level" => 16, "abbreviation" => "R"
 GradeLevel.create "title" => "College 5th-Year", "level" => 17
 
+# Help Texts
+HelpText.create({
+  "object_class" => "Participant", 
+  "attribute_name" => "household_size",
+  "hint" => "Include yourself in this count",
+  "audience" => "Participant"
+})
+HelpText.create({
+  "object_class" => "Mentor", 
+  "attribute_name" => "household_size",
+  "hint" => "Include the student in this count",
+  "audience" => "Mentor"
+})
+HelpText.create({
+  "object_class" => "Participant",
+  "attribute_name" => "immigrant",
+  "title" => "Do you identify yourself as an immigrant to the United States?",
+  "audience" => "Participant"
+})
+HelpText.create({
+  "object_class" => "Mentor",
+  "attribute_name" => "immigrant",
+  "title" => "Identifies as an immigrant to the United States",
+  "audience" => "Mentor"
+})
+HelpText.create({
+  "object_class" => "Participant",
+  "attribute_name" => "first_generation",
+  "title" => "Does one or more of your parents/legal guardians hold a bachelor’s/4 year degree?",
```

Comment:
Looking at the formtastic source, I see why this is happening. Those "choice" style inputs use a slightly different method for outputting the view, because the "label" is actually contained in a `<legend>` element instead. Take a look at https://github.com/justinfrench/formtastic/blob/master/lib/formtastic/inputs/radio_input.rb#L135 and you'll see the `legend_html` in use.

Looks like an easy solution would be to override `Formtastic::Inputs::Base::Choices#legend_html` just like we do the others. Want to give that a shot?